test/evaluation/evaluationClientWrapper.spec.ts

```diff
@@ -224,25 +224,25 @@ describe('evaluate', () => {
 
   it('should return error if all the uris in vp don\'t match at least one of input_descriptor\'s uris', function() {
     const pdSchema: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
-    vpSimple.verifiableCredential[0][`@context`] = 'https://www.w3.org/TR/vc-data-model/#types1';
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
+    vpSimple.verifiableCredential[0][`@context`] = ['https://www.w3.org/TR/vc-data-model/#types1'];
     const evaluationClientWrapper: EvaluationClientWrapper = new EvaluationClientWrapper();
     const evaluationClient: EvaluationClient = evaluationClientWrapper.getEvaluationClient();
     vpSimple.holder = HOLDER_DID;
-    const evaluationResults = evaluationClientWrapper.evaluate(pdSchema, new VP(vpSimple));
+    const evaluationResults = evaluationClientWrapper.evaluate(pdSchema, vpSimple);
     const errorResults = evaluationClient.results.filter(result => result.status === Status.ERROR);
     expect(errorResults.length).toEqual(2);
     expect(evaluationResults).toEqual(error);
   });
 
   it('should return ok if all the uris in vp match at least one of input_descriptor\'s uris', function() {
     const pdSchema: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.